benchmarks/htap/lib/stats.py

```diff
@@ -27,9 +54,7 @@ class Stats:
     the end state of a transaction is sent through the queue. Per reporting interval
     all accumulated events are sent, which contain the timestamp, query id, status, and runtime
     for each transaction.
-    This is kept in a list where the last HISTORY_LENGTH seconds of samples are stored.
-    When we want to report the actual statistics we then simply aggregate the results
-    based on the raw, possibly filtered, samples.
+    This is kept in a list where the last history_length seconds of samples are stored.
```

Comment:
but you're storing now a list of aggregated results, not samples? or?

Reply:
aggregated results, changed